File: python/001.two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        if len(nums)<2:
            return None
        map={}
        for i,n in enumerate(nums):
            if target-n in map:
                return [i,map[target-n]]
            map[n]=i

File: C++/001. Two Sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        if(nums.size()<2)return {};
        map<int,int> m;
        for(int i=0;i<nums.size();++i){
            if(m.find(target-nums[i])!=m.end())
                return {m[target-nums[i]],i};
            m[nums[i]]=i;
        }
        return {};
    }
};